modules/cache-common/src/internalClusterTest/java/org/opensearch/common/cache/tier/TieredSpilloverCacheBaseIT.java

```diff
@@ -6,7 +6,7 @@
  * compatible open source license.
  */
 
-package org.opensearch.cache.common.tier;
+package org.opensearch.common.cache.tier;
```

Comment:
Can we avoid renaming these packages? This has introduced a lot of changes.
Plus for `cache-common` module, it makes sense to keep package name prefix as `org.opensearch.cache`

Reply:
We can do this if we make a couple methods in `DefaultCacheStatsHolder` public. Otherwise I'm not sure the tests can work out. Will make these changes.